### DISCUSSION TITLE
Webpack 4::  minimize : false not working

### DISCUSSION THREAD

**Discussion by digiperfect-tech** [webpack/webpack#15026] [Q&A]
**Do you want to request a *feature* or report a *bug*?**

*bug*

**What is the current behavior?**

I am using a multiple exports setup, with `{ optimization : { minimize : false } }` in the configuration option. However, the output is still minified.

**If the current behavior is a bug, please provide the steps to reproduce.**

Following is the configuration object that I print just before pushing to webpack exports:
```


{
  mode: 'development',
  stats: {
    all: false,
    errors: true,
    warnings: true,
    moduleTrace: true,
    errorDetails: true
  },
  resolve: { extensions: [ '.tsx', '.ts', '.js', '.scss' ] },
  node: { __dirname: false },
  watchOptions: {
    ignored: [
      /docs/,
      /assets/,
      /.git/,
      'build/server.build.js',
      /scripts/,
      /node_modules/,
      /\.d\.ts$/
    ]
  },
  target: 'web',
  name: 'AppName',
  entry: './src/folderName/exports.tsx',
  optimization: { minimize: false },
  module: { rules: [ [Object], [Object], [Object] ] },
  externals: [
    {
      react: 'React',
      'react-dom': 'ReactDOM'
    }
  ],
  plugins: [
    DefinePlugin { definitions: [Object] },
    MiniCssExtractPlugin { options: [Object] },
    OptimizeCssAssetsWebpackPlugin {
      pluginDescriptor: [Object],
      options: [Object],
      phaseAssetProcessors: [Object],
      deleteAssetsMap: {}
    },
    _class {
      nodemonOptions: [Object],
      isWebpackWatching: false,
      isNodemonRunning: false
    }
  ],
  output: {
    library: 'LIbraryName',
    libraryTarget: 'var',
    path: '......some_path___.....',
    filename: 'exports.js',
    globalObject: "(typeof self !== 'undefined' ? self : this)"
  }
}



```

**What is the expected behavior?**

Exported output should be non minified

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, webpack version, and Operating System.**

Node version: v14.18.1
OS: Ubuntu 20.04
Webpack version: 4.45.0

**Comment by alexander-akait** [comment]
It means you still have minimizer plugin, check your configuration

**Reply by digiperfect-tech** [reply]
Hello @alexander-akait   

The plugins block looks like the following. Do you mean the mini css / optimize css assets plugins could also be minifying javascript? 

```
 plugins: [
    DefinePlugin { definitions: [Object] },
    MiniCssExtractPlugin { options: [Object] },
    OptimizeCssAssetsWebpackPlugin {
      pluginDescriptor: [Object],
      options: [Object],
      phaseAssetProcessors: [Object],
      deleteAssetsMap: {}
    },
    _class {
      nodemonOptions: [Object],
      isWebpackWatching: false,
      isNodemonRunning: false
    }
  ]
```

**Reply by alexander-akait** [reply]
No, `OptimizeCssAssetsWebpackPlugin` should not minimizer, can you provide original configuration? Note some libraries and packages can be minimized, webpack outputs their code as is
